apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/UpcomingInvoice.tsx

```diff
@@ -37,83 +71,161 @@ const UpcomingInvoice = ({ slug }: UpcomingInvoiceProps) => {
       {isError && <AlertError subject="Failed to retrieve upcoming invoice" error={error} />}
 
       {isSuccess && (
-        <table className="w-full">
-          <thead>
-            <tr className="border-b">
-              <th className="py-2 font-normal text-left text-sm text-foreground-light w-1/2">
-                Item
-              </th>
-              <th className="py-2 font-normal text-left text-sm text-foreground-light">Count</th>
-              <th className="py-2 font-normal text-left text-sm text-foreground-light">
-                Unit price
-              </th>
-              <th className="py-2 font-normal text-right text-sm text-foreground-light">Price</th>
-            </tr>
-          </thead>
-          <tbody>
-            {fixedFees.map((item) => (
-              <tr key={item.description} className="border-b">
-                <td className="py-2 text-sm">{item.description ?? 'Unknown'}</td>
-                <td className="py-2 text-sm">{item.quantity}</td>
-                <td className="py-2 text-sm">
-                  {item.unit_price === 0 ? 'FREE' : `$${item.unit_price}`}
-                </td>
-                <td className="py-2 text-sm text-right">${item.amount}</td>
+        <div>
+          {feesWithBreakdown.length && (
```

Comment:
```suggestion
          {feesWithBreakdown.length > 0 && (
```

There's a "0" rendered here
![image](https://github.com/supabase/supabase/assets/19742402/10aa4c66-4733-428a-b4b1-d9c7ecedcba1)